config/config.go

```diff
@@ -33,7 +33,6 @@ type Chain struct {
 	ResyncFrom              uint64
 	SkipBlocks              map[flow.Identifier]struct{}
 	SporkSealTolerance      uint64
-	SporkSyncedTolerance    uint64
```

Comment:
Why do we want to remove this?  Do we need this param to be flexible for handling spork syncing?

Reply:
never mind, you answered this in the later comment. :)